android/src/main/java/com/pagopa/ioreactnativecrypto/IoReactNativeCryptoModule.kt

```diff
@@ -647,4 +698,15 @@ fun ByteArray.base64NoWrap(): String {
   return Base64.encodeToString(this, Base64.NO_WRAP)
 }
 
+/**
+ * Encodes the byte array as a base64url string without padding,
+ * removing the leading 0x00 byte if present (to ensure unsigned integer representation).
+ */
+fun ByteArray.toBase64UrlUIntBytes(): String {
+  val sanitized = if (this.isNotEmpty() && this[0].toInt() == 0) {
+    this.copyOfRange(1, this.size)
+  } else this
+  return Base64.encodeToString(sanitized, Base64.URL_SAFE or Base64.NO_PADDING or Base64.NO_WRAP)
```

Comment:
We're not validating that EC key coordinates are exactly 32 bytes. Should we enforce that explicitly to avoid edge cases? What do you think?

Reply:
To improve clarity, I’ve implemented two separate extension functions: one specifically for EC keys that enforces the 32-byte constraint strictly, and another for RSA keys where that check is not applicable. Thanks for pointing it out!